src/ResourceManager/Sql/Commands.Sql/ServerDnsAlias/Cmdlet/GetAzureSqlServerDnsAlias.cs

```diff
@@ -0,0 +1,78 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Sql.ServerDnsAlias.Model;
+using System.Collections.Generic;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Sql.ServerDnsAlias.Cmdlet
+{
+	/// <summary>
+	/// Defines the Get-AzureRmSqlServerDnsAlias cmdlet
+	/// </summary>
+	[Cmdlet(VerbsCommon.Get, "AzureRmSqlServerDnsAlias", SupportsShouldProcess = true)]
+	[OutputType(typeof(Model.AzureSqlServerDnsAliasModel))]
+	public class GetAzureSqlServerDNSAlias : AzureSqlServerDnsAliasCmdletBase
+	{
+		/// <summary>
+		/// Gets or sets the name of the Azure Sql Server DNS Alias
+		/// </summary>
+		[Parameter(Mandatory = false,
+			HelpMessage = "The Azure Sql Server DNS Alias name.")]
+		[Alias("Name")]
+		[ValidateNotNullOrEmpty]
+		public string DnsAliasName { get; set; }
```

Comment:
@maboja-msft the naming should be swapped - the parameter name should be `Name` and the alias should be `DnsAliasName` (this is for consistency with other cmdlets)